iscsi/targetd/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
@wongma7 I'm not sure how else to contact you.
Are we ready to merge? Am I missing something?